proto/source.proto

```diff
@@ -11,6 +9,11 @@ message StreamResponse {
 	bytes output = 1;
 }
 
+message StreamRequest {
+	// Holds list of config YAML definitions in order specified by user
```

Comment:
What does this comment mean?

Reply:
It means that the `configs` property holds the list of config in YAML format, I also wanted to express that the order is preserved. 

The Go models are later auto-generated and this comment is also above given property so when you use this struct you know what to expect from it.